src/commands/init/index.ts

```diff
@@ -13,6 +13,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { Command, Flags } from '@oclif/core';
+import { ux } from '@oclif/core/lib/cli-ux';
```

Comment:
could you adapt either this or the import of "CliUx" in commands/upgrade/index.ts to be consistent?

Reply:
Adapted it here, auto import made a mess of this.